DuckDuckGo/RemoteMessaging.swift

```diff
@@ -179,7 +175,6 @@ struct RemoteMessaging {
                                                            favoritesCount: favoritesCount,
                                                            appTheme: AppUserDefaults().currentThemeName.rawValue,
                                                            isWidgetInstalled: isWidgetInstalled,
-                                                           isNetPWaitlistUser: isNetworkProtectionWaitlistUser,
                                                            daysSinceNetPEnabled: daysSinceNetworkProtectionEnabled),
```

Comment:
Do I need to remove this too, @samsymons ?

Reply:
Let's keep these since they'll be used for RMF attribute matching on Privacy Pro subscribers

Reply:
I want to make sure I understand correctly here.

I was asking about `daysSinceNetPEnabled` but now I wonder if your answer includes `isNetPWaitlistUser` too.

Reply:
No, we no longer need `isNetPWaitlistUser`, it's just `daysSinceNetPEnabled` that will be needed going forward.